src/ui/window.ui

```diff
@@ -1,279 +1,153 @@
     <child type="titlebar">
       <object class="GtkHeaderBar" id="headerbar">
-        <property name="visible">True</property>
-        <property name="can-focus">False</property>
-        <property name="title">Curtail</property>
-        <property name="subtitle" translatable="yes">Compress your images</property>
-        <property name="show-close-button">True</property>
         <child>
-          <object class="GtkButtonBox">
-            <property name="visible">True</property>
-            <property name="can-focus">False</property>
-            <property name="homogeneous">True</property>
-            <property name="layout-style">expand</property>
+          <object class="GtkBox">
+            <style>
+              <class name="linked"/>
+            </style>
             <child>
               <object class="GtkButton" id="back_button">
-                <property name="visible">True</property>
-                <property name="sensitive">False</property>
-                <property name="can-focus">True</property>
-                <property name="receives-default">True</property>
                 <property name="action-name">win.back</property>
-                <property name="image">back_image</property>
-                <property name="always-show-image">True</property>
+                <property name="icon-name">go-previous-symbolic</property>
               </object>
-              <packing>
-                <property name="expand">True</property>
-                <property name="fill">True</property>
-                <property name="position">0</property>
-              </packing>
             </child>
             <child>
               <object class="GtkButton" id="forward_button">
-                <property name="visible">True</property>
-                <property name="sensitive">False</property>
-                <property name="can-focus">True</property>
-                <property name="receives-default">True</property>
                 <property name="action-name">win.forward</property>
-                <property name="image">forward_image</property>
-                <property name="always-show-image">True</property>
+                <property name="icon-name">go-next-symbolic</property>
               </object>
-              <packing>
-                <property name="expand">True</property>
-                <property name="fill">True</property>
-                <property name="position">1</property>
-              </packing>
             </child>
           </object>
-          <packing>
-            <property name="position">3</property>
-          </packing>
         </child>
-        <child>
+        <child type="end">
           <object class="GtkMenuButton" id="menu_button">
-            <property name="visible">True</property>
-            <property name="can-focus">True</property>
-            <property name="receives-default">True</property>
             <child>
               <object class="GtkImage">
```

Comment:
Never set a GtkImage as a child of a GtkButton or GtkMenuButton, use the icon-name property. Otherwise the css is wrong.